CMakeLists.txt

```diff
@@ -1,3 +1,17 @@
+# This is the main CMake file for fv3atm.
+#
+# Dusan Jovic, Alex Richert
+cmake_minimum_required(VERSION 3.19)
```

Comment:
A quick question, with line 4, do we still need line 17?

Reply:
Good catch. I have removed line 17.